src/main/java/de/dennisguse/opentracks/AbstractTrackDeleteActivity.java

```diff
@@ -69,6 +93,8 @@ protected void deleteTracks(Track.Id... trackIds) {
     public void onConfirmDeleteDone(Track.Id... trackIds) {
```

Comment:
Why not pass a list?

Reply:
I didn't want to touch it.

Reply:
This should be fixed/changed in other separated issue because it affects a lot of parts: `Track.Id... trackIds` are everywhere.